DQMOffline/Trigger/plugins/LepHTMonitor.cc

```diff
@@ -0,0 +1,488 @@
+#include "DQMOffline/Trigger/plugins/LepHTMonitor.h"
+
+#include <limits>
+#include <algorithm>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/Common/interface/TriggerNames.h"
+
+#include "DataFormats/HLTReco/interface/TriggerObject.h"
+
+#include "DataFormats/Math/interface/deltaR.h"
+
+#include "DataFormats/EgammaCandidates/interface/ConversionFwd.h"
+#include "DataFormats/EgammaCandidates/interface/Conversion.h"
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "RecoEgamma/EgammaTools/src/ConversionTools.cc" //Would prefer to include header, but fails to find hasMatchedConversion definition without this..
+#include "DataFormats/PatCandidates/interface/Muon.h"
+
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+namespace{
+
+  //Offline electron definition
+  bool IsGood(const reco::GsfElectron &el, const reco::Vertex::Point &pv_position,
```

Comment:
hi @heller3 - by convention, functions should start with a lower case letter - eg, isGood.